Doc/source/designspaceLib/index.rst

```diff
@@ -1,16 +1,16 @@
-##############
-designspaceLib
-##############
+########################################################
+designspaceLib: Read, write, and edit .designspace files
+########################################################
 
 MutatorMath started out with its own reader and writer for designspaces.
 Since then the use of designspace has broadened and it would be useful
 to have a reader and writer that are independent of a specific system.
```

Comment:
This should document what the module actually does. The reader might interpret “would be useful” to mean the developers plan to implement it but haven’t gotten to it yet.

Reply:
Good point, and the MutatorMath reference isn't necessarily relevant any more.